pkg/controller/bastion/options.go

```diff
@@ -196,3 +193,45 @@ func FirewallEgressAllowOnlyResourceName(baseName string) string {
 func FirewallEgressDenyAllResourceName(baseName string) string {
 	return fmt.Sprintf("%s-deny-all", baseName)
 }
+
+func getCloudProfileConfig(cluster *extensions.Cluster) (*gcpv1alpha1.CloudProfileConfig, error) {
```

Comment:
Maybe this can be replaced with https://github.com/gardener/gardener-extension-provider-gcp/blob/07acf111fa7321939d63be6c36597cdacde0895c/pkg/apis/gcp/helper/scheme.go#L66 ?